cli/src/args.rs

```diff
@@ -17,6 +17,14 @@ pub struct CliArguments {
     /// The typst command to run
     #[command(subcommand)]
     pub command: Command,
+
+    /// Sets the level of verbosity: 0 = warning & error, 1 = info, 2 = debug, 3 = trace
+    #[clap(short, long, action = ArgAction::Count)]
+    pub verbosity: u8,
+
+    /// Whether to enable debug mode
+    #[clap(long)]
+    pub debug: bool,
```

Comment:
I'm not sure about`--debug` and `backtrace()`. Can we remove it from this PR and consider it separately?

Reply:
Sure! I will (for now) still keep the ability to create backtraces (internally) using tracing as it can be useful for debugging, but I'll move it to a rust-only function in `utils`